recipes/pygenetic_code/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pygenetic_code" %}
+{% set version = "0.20.0" %}
+{% set sha256 = "2d4773e4e8a8b994e604b986ce831b12bbf68b41780e24377bfc14639e017423" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: '{{ version }}'
+
+source:
+  url: "https://files.pythonhosted.org/packages/b6/6f/034dc48dd7768cb5aecabb4c2b68e5ff48d7fea6ee3fc51fed7a068e7c11/{{ name }}-{{ version }}.tar.gz"
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv "
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x") }}
+    
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python >=3
+    - scikit-build-core
+    - pip
+    - make
+    - cmake >=3.15
+  host:
+    - python >=3
+    - scikit-build-core
+    - zlib
+  run:
+    - python >=3
+    - zlib
```

Comment:
can be removed as run dependency, because it carries a run_export within itself.